### PR TITLE
fix: no connection dataloaders for django 2

### PR DESCRIPTION
**Pull request by superlevure**

### CODE REVIEW

File: graphene_django/relay/connection.py [hunk 1]
```diff
@@ -305,3 +294,18 @@ def _handle_last_before(
...
+        and graphene_settings.USE_DATALOADERS
+        and isinstance(sized_sliceable, QuerySet)
+        and info is not None
+        and hasattr(info, "context")
+        and hasattr(info.context, "dataloaders")
+        and isinstance(info.context.dataloaders, dict)
+    ):
+        return True
+
+    return False
```
Comment (lucas-bremond) [inline graphene_django/relay/connection.py:311]: Why not simply:

```suggestion
    return (
        django.VERSION[0] >= 3
        and graphene_settings.USE_DATALOADERS
        and isinstance(sized_sliceable, QuerySet)
        and info is not None
        and hasattr(info, "context")
        and hasattr(info.context, "dataloaders")
        and isinstance(info.context.dataloaders, dict)
    )
```

File: graphene_django/relay/connection.py [hunk 2]
```diff
@@ -305,3 +294,18 @@ def _handle_last_before(
         has_previous_page,
         has_next_page,
     )
+
+
+def _use_connection_dataloaders(info, sized_sliceable):
```
Comment (lucas-bremond) [inline graphene_django/relay/connection.py:299]: Curious to know why we've reverted back to a succession of progressive `hasattr`.

Wasn't the 

```py
try:
    dataloaders_context = info.context.dataloaders
except AttributeError:
    pass
```

+ later checks more optimal?

File: graphene_django/relay/connection.py [hunk 3]
```diff
@@ -305,3 +294,15 @@ def _handle_last_before(
...
+    return (
+        django.VERSION[0] >= 3
+        and graphene_settings.USE_DATALOADERS
+        and isinstance(sized_sliceable, QuerySet)
+        and info is not None
+        and hasattr(info, "context")
+        and hasattr(info.context, "dataloaders")
+        and isinstance(info.context.dataloaders, dict)
+    )
```
Comment (lucas-bremond) [inline graphene_django/relay/connection.py:308]:
```suggestion
    if (
        django.VERSION[0] >= 3
        and graphene_settings.USE_DATALOADERS
    ):
        try:
            return (
                isinstance(info.context.dataloaders, dict)
                and isinstance(sized_sliceable, QuerySet)
            )
        except AttributeError:
            return False
            
return False
```

File: graphene_django/relay/connection.py (hunk 2 above)
Comment (lucas-bremond) [reply graphene_django/relay/connection.py:299]: Added suggestion below

File: graphene_django/relay/connection.py [hunk 4]
```diff
@@ -305,3 +294,18 @@ def _handle_last_before(
         has_previous_page,
         has_next_page,
     )
+
+
+def _use_connection_dataloaders(info, sized_sliceable):
+    if (
```
Comment (tcleonard) [inline graphene_django/relay/connection.py]: why not simply something like:
```python
return (
    django.VERSION[0] >= 3
    and graphene_settings.USE_DATALOADERS
    ...
)
```

and remove the if statement

Comment (lucas-bremond) [reply graphene_django/relay/connection.py]: LOL too late @tcleonard

File: graphene_django/relay/connection.py (hunk 2 above)
Comment (superlevure) [reply graphene_django/relay/connection.py:299]: applied, thanks

**Review by lucas-bremond — APPROVED** [review]

**Pull request merged**